docs/src/algorithms/discretization.md

```diff
@@ -5,3 +5,39 @@ discretize
 DiscretizationMethod
 FIST
 ```
+# FIST  
+Examples for the FIST algorithm. 
+  
+
+<h1>ABCD</h1>
+<p markdown="1">
+asdfhibsfdifgsdf
+sdffjksndjofnwsdf
+</p>
```

Comment:
Also, please pay attention to these dead code when you submit PRs. Be 100% focused when you are working on these contributions.

Reply:
I'm so sorry, I'll remove that immediately.